websocket-client-patches/README.md

```diff
@@ -0,0 +1,32 @@
+# websocket-client wolfSSL port
+
+This folder contains patches to add support for wolfSSL to websocket-client project.
+Subfolder x.yy.zz contains patches for urllib3 version x.yy.zz.
```

Comment:
urllib3 should be websocket-client here?

Reply:
Thanks! I also add a mention to use WEBSOCKET_CLIENT_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt in the README.md